runvx/vxUtils.cpp

```diff
@@ -571,6 +571,43 @@ int WriteImage(vx_image image, vx_rectangle_t * rectFull, FILE * fp)
 	return 0;
 }
 
+// write image compressed
+int WriteImageCompressed(vx_image image, vx_rectangle_t * rectFull, const char * fileName) {
+    // get number of planes, image width in bytes for single plane
```

Comment:
Keep '{' at the beginning of line to make the code look similar to other functions.

Reply:
Done.